src/lib/src/footer/footer.component.ts

```diff
@@ -9,12 +9,7 @@ import { Component } from '@angular/core';
       <div class="columns">
         <section>
           <div class="beta-notice">
```

Comment:
might make sense to change the class name now

Reply:
Good call.